CONTRIBUTION.md

```diff
@@ -0,0 +1,184 @@
+    * **Explain why this enhancement would be useful**.
+* Give your **acceptance criteria**
+    * What behaviour do you want to see
+    * What are your minimum requirements
+* The **Definition of Done** is already predefined by us, but you can add points when you feel like something is missing. 
+
+### Your First Code Contribution
+
+When you first contribute to Antenna, it can be that it takes a while for us to start reviewing your code. 
+Pleas be patient, and if necessary, write one of the contributors and ask what the state is on your PR. 
+
+Be sure you fulfill all legal requirements (e.g. [Eclipse Contributor Agreement](#eclipse-contributor-agreement)), otherwise we can't accept your pull request. 
+
+### Pull Requests
+
+The process described here has several goals:
+
+- Maintain Antenna's quality
+- Enable a sustainable system for maintainers to review contributions
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. Describe in your PR what you have done, what this is fixing or introducing and why this isn't regressive. 
+If there is an issue to your PR, please link it. 
+2. Follow the [styleguides](#styleguides).
+3. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing.
+Should a status check fail and you believe it is unrelated to your contribution, please write so in a comment in the PR. 
+
+While the prerequisites above may be satisfied, the reviewer(s) may still ask you to do additional work or ask for changes before your PR is finally accepted. 
+Since the reviewer knows the code very well, they might have ideas how to include features more elegantly or see inconsistencies compared to the code base. 
+
+## Styleguides
+
+### Git Commit Messages
+
+We use the commit style of the [SW360 project](https://github.com/eclipse/sw360/wiki/Dev-Semantic-Commits).
+
+### Java Styleguide
+
+All Java should adhere to the [Java Language Specification](https://docs.oracle.com/javase/specs/).
+
+### Documentation Styleguide
+
+Our documentation mainly focuses on the usage of Antenna. 
+So should you introduce any new properties, configurations, workflow steps or anything else the user will have to configure, please add a documentation about this. 
+Also, when you introduce path sensitive features or changes (like reading in a document), make sure it works with most standard OSs Antenna supports.
+ Any drawbacks, additional information and small notes should also be documented.
+ 
+ For our documentation we use [Markdown](https://daringfireball.net/projects/markdown) and the [Doxia Markup Languages References](https://maven.apache.org/doxia/references/index.html). 
+ With the Doxia Markup Language References we reference the name of the product and some other values that can be found in the site modules pom file. 
+
+Our documentation is written in English. 
```

Comment:
Please clarify whether British or American English (I believe it should be American English).